src/neo-vm/ExecutionEngine.cs

```diff
@@ -378,45 +366,16 @@ private bool ExecuteInstruction()
                         if (!context.EvaluationStack.TryRemove(n, out StackItem _)) return false;
                         break;
                     }
-                case OpCode.XSWAP:
+                case OpCode.CLEAR:
```

Comment:
Does we need another CLEAR for alt stack?

Reply:
AltStack is never needed and will be removed when slot opcodes are applied.